src/routes/info.tsx

```diff
@@ -0,0 +1,62 @@
+import { Handler, t } from 'elysia';
+import {
+  dependencies as deps,
+  version
+} from '../../package.json';
+
+export const versionsApi = {
+  response: t.Any({
+    description: 'Information about the installed versions of the GeoStyler parsers.' +
+      ' If the `Accept` header is set to `application/json`, the response will be a JSON object. ' +
+      'Otherwise, the response will be an HTML page.'
+  })
+};
+
+export const versions: Handler = ({
+  request
+}) => {
+  if (request.headers.get('accept') === 'application/json') {
+    return {
+      'geostyler-rest': version,
+      'geostyler-mapbox-parser': getVersionString('geostyler-mapbox-parser'),
+      'geostyler-mapfile-parser': getVersionString('geostyler-mapfile-parser'),
+      'geostyler-qgis-parser': getVersionString('geostyler-qgis-parser'),
+      'geostyler-sld-parser': getVersionString('geostyler-sld-parser'),
+      'geostyler-lyrx-parser': getVersionString('geostyler-lyrx-parser')
```

Comment:
A stylistic thing: I would have created the object outside of the `if`, and used it both branches (returned or used to construct HTML)